gdp/column/gdp_column.py

```diff
@@ -45,180 +68,147 @@ def build_column(min_trays, max_trays, xD, xB):
                     'D': 0, 'E': 0},
         'toluene': {'A': 1.40E5, 'B': -1.52E2, 'C': 6.95E-1,
                     'D': 0, 'E': 0}}
-    m.dH_vap = {'benzene': 33.770E3, 'toluene': 38.262E3}  # J/mol
+    m.dH_vap = {'benzene': 33.770E3, 'toluene': 38.262E3}  # Enthaply for vaporation [J/mol]
```

Comment:
Make sure that these are the correct units

Reply:
I changed the flow rate [kmol] into [kmol/s].

Reply:
and back to [mol/s]